core/tests/ledger_cleanup.rs

```diff
@@ -275,6 +295,27 @@ mod tests {
         *data_shred_storage_previous = data_shred_storage_now;
     }
 
+    /// Helper function of the benchmark `test_ledger_cleanup_compaction` which
+    /// returns true if the benchmark fails the size limitation check.
+    fn is_exceeded_stop_size_iterations(
+        storage_size: u64,
+        stop_size: u64,
+        exceeded_iterations: &mut u64,
+        iteration_limit: u64,
+    ) -> bool {
+        info!("is_exceeded_stop_size_iterations {} {} {} {}", storage_size, stop_size, &exceeded_iterations, iteration_limit);
```

Comment:
I admittedly haven't run with this change yet, but wondering if this print is too noisy? We know the check will happen if we specify those values, and I think we only care to be alerted if we exceed, right? Also, the `emit_stats()` will have the shred storage I believe so we can already track that value from those prints.